editor/test/resources/game_project_properties/game.project

```diff
@@ -1,6 +1,6 @@
 [project]
 title = Game Project Properties
-dependencies = http://test.com/test.zip
+dependencies = http://test.com/not-responding.zip
```

Comment:
We have a test that verifies that non-responding dependencies are not included in the build output, so we want to keep a URL here that can be resolved but will eventually time out. There doesn't appear to be an "official" test URL that guarantees this behavior, but I changed the zip file name for clarity.